source/extensions/common/wasm/context.cc

```diff
@@ -1108,9 +1108,9 @@ StreamInfo::StreamInfo* Context::getRequestStreamInfo() const {
 
 const Network::Connection* Context::getConnection() const {
   if (encoder_callbacks_) {
-    return encoder_callbacks_->connection();
+    return encoder_callbacks_->connection().ptr();
```

Comment:
in a follow-up will you change getConnection() to also return an OptRef?

Reply:
maybe?  :-)